examples/jsm/nodes/materials/Line2NodeMaterial.js

```diff
@@ -368,7 +369,9 @@ class Line2NodeMaterial extends NodeMaterial {
 					const instanceColorStart = attribute( 'instanceColorStart' );
 					const instanceColorEnd = attribute( 'instanceColorEnd' );
 
-					lineColorNode = varying( positionGeometry.y.lessThan( 0.5 ).cond( instanceColorStart, instanceColorEnd ) );
+					const instanceColor = positionGeometry.y.lessThan( 0.5 ).cond( instanceColorStart, instanceColorEnd );
+
+					lineColorNode = color( instanceColor ).mul( color( materialColor ) );
```

Comment:
Nodes converts types automatically if necessary, I think `lineColorNode = instanceColor.mul( materialColor )` should work.